2020/hamming-codes/spanish/sentence_translations.json

```
@@ -479,7 +479,7 @@
  {
   "input": "And actually, we should be clear, if the receiver sees an odd parity, it doesn't necessarily mean there was just one error, there might have been 3 errors, or 5, or any other odd number, but they can know for sure that it wasn't 0.",
   "model": "nmt",
-  "translatedText": "Y, de hecho, debemos ser claros: si el receptor ve una paridad impar, no significa necesariamente que hubo solo un error, puede haber habido 3 errores, o 5, o cualquier otro número impar, pero pueden estar seguros. que no era 0.",
+  "translatedText": "Y, de hecho, debemos ser claros, si el receptor ve una paridad impar, no significa necesariamente que hubo solo un error, puede haber habido 3 errores, o 5, o cualquier otro número impar, pero puede estar seguro de que no era 0.",
```

I think a colon fits better the structure of the sentence (If I am not mistaken, I think it would be an example of [2.6 in this definition](https://www.rae.es/dpd/dos%20puntos)). On the other hand, if the "0" refers to "0 errors", I think a "fueron" sounds better.

```suggestion
  "translatedText": "Y, de hecho, debemos ser claros: si el receptor ve una paridad impar, no significa necesariamente que hubo solo un error, puede haber habido 3 errores, o 5, o cualquier otro número impar, pero puede estar seguro de que no fueron 0.",
```